src/os/bluestore/BlueStore.cc

```diff
@@ -6147,6 +6150,131 @@ void BlueStore::_wctx_finish(
   o->onode.compress_extent_map();
 }
 
+
+bool BlueStore::blobs_need_garbage_collection(
+  OnodeRef o,
+  uint64_t start_offset, 
+  uint64_t end_offset,
+  uint32_t *blob_depth,
+  uint64_t *gc_start_offset, 
+  uint64_t *gc_end_offset)
+{
+  uint32_t depth = 0;
+  bool head_overlap = false;
+  bool tail_overlap = false;
+
+  *gc_start_offset = start_offset;
+  *gc_end_offset   = end_offset;
+
+  map<uint64_t,bluestore_lextent_t>::iterator hp =
```

Comment:
Done the changes. (After a series of changes, the final code ended up  being the duplicate of seek_lextent).